### PR TITLE
Cow Bull game

### LINKED ISSUE #4799: [New game]:  Cow Hunting
### 🎮 Game Request

User need to tap on the cow so that the point will get increase

### Point down the features

It will be very interesting coz of the speed of cow will get increase when user get high points

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by mukilan2815**
## PR Description 📜

Fixes #4799 

<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [x] I have added working video of the game in README.md (optional)
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸

image

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @mukilan2815 ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

File: README.md
Comment (ishita-43) [inline README.md:1]: pLease sync your readme file with latest changes  and only add than your game entry in there

**Review by ishita-43 — CHANGES_REQUESTED** [review]
Please make the needed changes in your PR

**Comment by ishita-43** [comment]
Please resolve merge conflicts

**Comment by mukilan2815** [comment]
oops sorry i will PR again

**Comment by mukilan2815** [comment]
done

**Comment by mukilan2815** [comment]
@ishita-43  #5020 i have created there and i will close here. please merge

**Pull request closed without merging**